app/src/pages/inside/common/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -260,6 +280,14 @@ export class Hamburger extends Component {
     } = this.props;
 
     const clusterTitle = this.getClusterTitle();
+    const canUpdateImportant = canDeleteLaunch(
+      accountRole,
+      projectRole,
+      this.props.userId === this.props.launch.owner,
+    );
+    const updateImportantTitle = canUpdateImportant
```

Comment:
Sounds like a function, may be `retentionTitle`?
Or even remove this variable and use the condition directly on component after refactoring.